aptos-move/move-examples/marketplace/sources/collection_offer.move

```diff
@@ -92,14 +91,33 @@ module collection_offer {
         item_price: u64,
         amount: u64,
         expiration_time: u64,
+    ) {
+        init_for_tokenv1_internal<CoinType>(
+            purchaser,
+            creator_address,
+            collection_name,
+            fee_schedule,
+            item_price,
+            amount,
+            expiration_time
+        );
+    }
+
+    public fun init_for_tokenv1_internal<CoinType>(
```

Comment:
_internal is a misleading name for a public function. I prefer "_entry" for the entry function and no suffix for the non-entry public function